crates/python/src/qpu/api.rs

```diff
@@ -187,34 +191,83 @@ pub struct ExecutionResults {
     /// QPU execution duration.
     #[pyo3(get)]
     pub execution_duration_microseconds: Option<u64>,
+    #[pyo3(get)]
+    pub memory: HashMap<String, Option<PyMemoryValues>>,
```

Comment:
Why is this `Option<PyMemoryValues>`? Shouldn't a `None` correspond to an entry not existing in the map?

Reply:
Oops, should've been cleaned up with the others, fixed.